modules/ROOT/pages/_partials/keycloak/setting-up-the-idm-sdk.adoc

```diff
@@ -80,10 +80,10 @@ Cordova::
 +
 [source,bash]
 ----
-$ npm install @aerogear/auth
+$ npm install @aerogear/auth --save
 ----
-. Use e.g `browserify` so that you can use the `require` statement
-. In the `init` snippet use `require` instead of `import`
+Use a JavaScript blunder e.g `browserify` so that you can use node modules in Cordova. We recommend the `require` statement rather that `import` for the use of the @aerogear/auth module as outline in the code snippet below.
```

Comment:
```suggestion
Use a JavaScript bundler, for example, `browserify` so that you can use node modules in Cordova. {org-name} recommends using the `require` statement rather than `import` for the the @aerogear/auth module as shown below:
```

Reply:
This comment is now marked as outdated. In addition, the update has been made and can be viewed here: https://github.com/aerogear/mobile-docs/pull/676/files